docs/standard/microservices-architecture/net-core-net-framework-containers/container-framework-choice-factors.md

```diff
@@ -43,17 +43,16 @@ There are several features of your application that affect your decision. You sh
     - Your .NET implementation choice is *.NET Framework* based on framework dependency.
     - Your container platform choice must be *Windows containers* because of the .NET Framework dependency.
 * Your application uses **SignalR services**.
-    - Your .NET implementation choice is *.NET Framework*, or *.NET Core (future release)*.
-    - Your container platform choice must be *Windows containers* because of the .NET Framework dependency.
-    - When **SignalR services** run on *.NET Core*, you can also choose *Linux containers*.
+    - Your .NET implementation choice is *.NET Framework*, or *.NET Core 2.1 (when released) or later*.
+    - Your container platform choice must be *Windows containers* if you chose the .NET Framework dependency.
```

Comment:
Is this still true with the compat shim for .NET Standard 2.0?

Reply:
The new compat shim in .NET Core 2.0 can be used to reference regular .NET Framework class libraries. But still, if a service, as a whole project, is using a framework like SignalR in .NET Framework or another example, Web Forms in .NET Framework, you need Windows Containers for that. 
I'm changing the sentences to the following, so it is clearer:

Your application uses **SignalR services**.
    - Your .NET implementation choice can be *.NET Framework*, or *.NET Core 2.1 (when released) or later*.
    - Your container platform choice must be *Windows containers* if you chose the SignalR implementation in .NET Framework.
    - Your container platform choice can be either Linux containers or Windows containers if you chose the SignalR implementation in .NET Core 2.1 or later (when released).